packages/clerk-js/src/core/clerk.ts

```diff
@@ -1345,4 +1347,28 @@ export default class Clerk implements ClerkInterface {
       throw new Error('ClerkJS components are not ready yet.');
     }
   }
+
+  shouldRedirectFAPIInitiatedFlow = async (): Promise<boolean> => {
```

Comment:
❓ @nikosdouvlis should we make this function private or do we consider a method that can be used in custom flows? I think it should be private.
```
#redirectFAPIInitiatedFlow  = async (): Promise<boolean> => {
```

Reply:
I agree that it should be private too. I will update the function declaration!